sklearn/datasets/openml.py

```diff
@@ -13,20 +13,68 @@
 
 
 import numpy as np
+import sklearn.externals.arff as arff
```

Comment:
is there a reason you import it here differently as the others (eg externals.joblib below as relative path)